db/tests/test_posts.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import db.posts as psts
+
+@pytest.fixture()
+def generate_post_fields():
+    return {
+        psts.USER_ID: None, 
```

Comment:
It should have the corresponding user_id when it gets posted

Reply:
fixed it.